### DISCUSSION TITLE
how i can change the default nvim ui command input type?

### DISCUSSION THREAD

**Discussion by m1guer** [stevearc/dressing.nvim#116] [Q&A] [answered]
when i press ":" the ui not shows only shows the default from nvim.
![image](https://github.com/stevearc/dressing.nvim/assets/91486755/02e85648-29cd-4a74-804b-bc892291a2e3)

**Comment by stevearc** [comment, marked as answer]
That is not what this plugin does. You might be looking for [noice.nvim](https://github.com/folke/noice.nvim)

**Reply by m1guer** [reply]
thx
